resources/wpt-prefs.json

```diff
@@ -1,4 +1,5 @@
 {
   "dom_webxr_test": true,
+  "layout_grid_enabled": true,
```

Comment:
It's quite weird to do this for only a single feature. We have many other features that are turned on only for our testing.

Reply:
Is there any reason to have features turned off for WPT runs? I mean, I guess if they cause crashes or something, but other than that we're just making ourselves look worse for no reason?

Reply:
Means the wpt.fyi results aren't very useful for feature flagged functionality. I wanted to compare Servo's grid support with Ladybird's as a guide to what to work on. But wpt.fyi just shows Servo as supporting almost nothing.

Reply:
It's also closer to reality. When you use Servo, all of unbaked features that are tuned off by default are not available to you unless you turn them on. We should finish feature implementation and turn features on by default.

Reply:
I share Martin's position.

Reply:
wpt.fyi is capable of running test for stable, beta and experimental versions of a browser.

Example: https://wpt.fyi/results/?label=master&product=chrome%5Bexperimental%5D&product=chrome%5Bbeta%5D&product=chrome%5Bstable%5D&product=servo%5Bexperimental%5D&product=servo%5Bbeta%5D&product=servo%5Bstable%5D

 - Chrome experimental is 135.0.7026.0 with --enable-experimental-web-platform-features
 - Chrome beta is 134.0.6998.15
 - Chrome stable is 133.0.6943.126
 - Servo experimental is 0.0.1-5ff5884
 - Servo beta and stable are non-existent.

Given that Servo is considered experimental, it's probably fine to enable experimental prefs just like Chrome (see #35576).
And we could also ask to have beta/stable runs without experimental prefs.

Reply:
In the spirit of compromise, this seems like a reasonable plan to me! Let's do that rather than enabling just a single preference here.